README.md

```diff
@@ -3,7 +3,7 @@
 <img src="./assets/pytrello2.png" width="160" height="160" align="right">
 
 [![CI
-Python](https://github.com/5-jigglypuff/pytrello2/actions/workflows/ci-tests.yml/badge.svg?branch=master)](https://github.com/5-jigglypuff/pytrello2/actions/workflows/ci-tests.yml)
+Python](https://github.com/5-jigglypuff/pytrello2/actions/workflows/tests.yml/badge.svg?branch=master)](https://github.com/5-jigglypuff/pytrello2/actions/workflows/tests.yml)
```

Comment:
Do you think we can open a different PR for this change?

Let's maintain the scope of this PR to add the two functions for board.

WDYT?

Reply:
Sounds good, I'll add a new PR for the readme.